spinalcordtoolbox/labels.py

```diff
@@ -267,6 +267,39 @@ def label_vertebrae(img: Image, vertebral_levels: Sequence[int] = None) -> Image
     return out
 
 
+def find_missing_label(img, ref):
```

Comment:
I'm not sure an entirely new function is necessary here, because it's not needed by anything else, is it?

 I think it might be better to focus on fixing the bug inside MSE, just so that things don't get overly complicated.

Reply:
I think that it could be useful. It could be of interest for some user, to know which labels are missing (check if it is systematic or not). It would be easier to retrieve the information than to use the 'Warning' in the MSE

Reply:
Thanks for the clarification! Your understanding of typical SCT usage is helpful here -- I'm glad to know there's a use case outside of MSE. :)